ui/components/List/List.vue

```diff
@@ -87,13 +81,14 @@ export default {
 
       this.data = data;
     },
-    openForm(name) {
+    reqOpenForm(name) {
       this.activeItem = name;
-      this.$router.push(`/edit/${this.doctype}/${name}`);
+      this.$emit("openForm",name);
     },
     async deleteCheckedItems() {
-      await frappe.db.deleteMany(this.doctype, this.checkList);
+      await frappe.db.deleteMany(this.doctype,this.checkList);
       this.checkList = [];
+      this.$emit('deleted',this.checkList);
```

Comment:
Move this above `this.checkList = []`.  Checklist should be cleared after emitting the event